runtime/compiler/runtime/JITServerAOTDeserializer.cpp

```diff
@@ -198,16 +198,16 @@ assertSharedVmAccess(J9VMThread *vmThread)
  *
  * @param records The serialization records in packed format (just a stream of bytes)
  * @param recordsSize The size of all the serialization records received
- * @param comp Compilation object
+ * @param context DeserializerHelper object
```

Comment:
DeserializationHelper --> DeserializationContext

Reply:
Fixed in force push.